src/my_ai.rs

```diff
@@ -35,17 +35,21 @@ impl MyAI {
 
     fn handle_about(&mut self, _cmd: &str) -> bool {
         let bot_name = "my AI";
+
         println!("name=\"{}\", version=\"1.0\", author=\"Nymand\", country=\"USA\"", bot_name);
         false
     }
 
     fn handle_start(&mut self, cmd: &str) -> bool {
         let parse: Vec<&str> = cmd.split_whitespace().collect();
-
         if let Some(&number_str) = parse.get(1) {
-            if let Ok(size) = number_str.parse::<usize>() {
-                self.my_board.resize(size);
-                self.my_board.size = size;
+        if let Ok(size) = number_str.parse::<usize>() {
+                if size != 20 {
+                    println!("ERROR invalid size.");
```

Comment:
This is not indented correctly, do you use the auto format ?

Reply:
no i do not